packages/components/src/query-controls/index.js

```diff
@@ -37,6 +36,16 @@ export default function QueryControls( {
 	onOrderChange,
 	onOrderByChange,
 } ) {
+	const categorySuggestions = categoriesList.reduce(
```

Comment:
This appears to be a controlled component so let's not make it handle any logic other than rendering. I'd suggest having `categorySuggestions` as a prop and moving this back to the `latestPosts` edit component?